code/processes/dqe.q

```diff
@@ -41,6 +44,7 @@ qpostback:{[proc;query;params;querytype;result]
   .lg.o[`qpostback;"Postback successful for ",string first proc];
   }
 
+/- function used to send queries to test processes
```

Comment:
I think use of "function used" in the comments is redundant, by context the user know this is a function.